components/docs/pluginsList.tsx

```diff
@@ -33,9 +34,16 @@ export default function PluginsList() {
 
     if (!response.objects) return;
 
+    const pluginDocsUrl = (pkg: { name: string }) => {
+      const name = pkg.name
+        .replace(/\@/g, "") // Characters to remove
+        .replace(/[\ \/]/g, "-"); // Characters to replace with a hyphen
+      return `/docs/plugins/${name}`;
+    };
+
     const packages: NPMPackage[] = response.objects
       .filter((o) => o.package)
-      .map((o) => o.package)
+      .map((o) => ({ ...o.package, url: pluginDocsUrl(o.package) }))
       .filter((o) => !ignoredPlugins.includes(o.name))
```

Comment:
We may want to hide some plugins too, like perhaps `app-templates` doesn't belong